### ISSUE TITLE
not all "lrzmax" are changed to setup0%lrzmax

### ISSUE THREAD

**Issue by ypetrov** [garrettwrong/cql3d_f90#33]
Looks like there are still many "lrzmax", rather than setup0%lrzmax.
In freyasou and freya.   Search "(lrzmax"

**Comment by garrettwrong** [comment]
yes, this is why I want test coverage and why I wanted `implicit none`.  That would catch these errors and more before they get into the code, so it is much safer to change and maintain...

Do you feel we have got them all, or do you have more?

**Comment by ypetrov** [comment]
I am not sure if I got them all.
In your editor, can you search for all "lrz", but excluding "%lrz",  
for example?
In my editor, if I try "lrz", whole-word option, it finds all those  
setup0%lrz instances.

On implicit: I can start introducing implicit none to most important  
subroutines, like those in netcdfrw2 and in impavnc.
Do you want me to work on this?

Yuri

Quoting Garrett Wright <notifications@github.com>:

**Comment by garrettwrong** [comment]
yes, whatever work you can do towards implicit none #2 is good.  I think next week we'll talk about what next etc.
